indico/web/client/js/react/components/Datepicker.jsx

```diff
@@ -0,0 +1,78 @@
+// This file is part of Indico.
```

Comment:
(I'm starting new threads here for anything I find while testing if I cannot map it to any specific location in the code, simply because it's nicer to reply to compared to top-level comments where GitHub unfortunately has no threading...)

The calendar always opens to the bottom, even if there is not enough space because the triggering element is close to the bottom edge of the page:

![image](https://github.com/indico/indico/assets/179599/1dd059c6-69ce-4bb9-bce7-2dfa9421e7bc)

Reply:
Which browser (and browser version) are you using? There should be a drop shadow behind the calendar dialog.

Reply:
Firefox 127.0b8 🦊

Reply:
`--control-raised-shadow` isn't set

Reply:
Yup. It was set on one of the other PRs, forgot to re-check after switching back to master.

Reply:
Did you mean `--modal-control-box-shadow`? I had a try and that looks much better!

![image](https://github.com/indico/indico/assets/179599/f9aafc73-e658-413d-b437-72d36a80134b)